automated install/basic-install.sh

```diff
@@ -351,37 +351,59 @@ package_manager_detect() {
         # Then check if dnf or yum is the package manager
         if is_command dnf ; then
             PKG_MANAGER="dnf"
-        else
+        fi
```

Comment:
I think you could go with an `elif` here.

Reply:
Correct, changed